script.js

```diff
@@ -1,9 +1,13 @@
-var display = document.getElementById("screen");
-var buttons = document.getElementsByClassName("button");
-
-Array.prototype.forEach.call(buttons, function (button) {
+let display = document.getElementById("screen");
+const buttons = Array.from(document.querySelectorAll(".button"));
+const backbtn=document.querySelector('.back');
+
+backbtn.addEventListener('click',()=>{

```

Comment:
if the button has the `button` class, move this to the other button initialisers.